### PR TITLE
Julius Dralle - Exercise 09(0.53634178089057)

### PR DESCRIPTION
**Pull request by RubelOS**
I had 0.53634178089057 at the very first run

I was able to reach a MC of 70% on both validating and training data

### CODE REVIEW

**Review by fabischw — COMMENTED** [review]
the commit message indicates this is the final state of the repo, please adjust the repo title to make it obvious this is no longer WIP (work in progress).

File: Exercises/JuliusDralle/exercises_09.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+classifier = tree.DecisionTreeClassifier(max_depth=8)
+classifier.fit(features_train, label_train)
+label_predicted = classifier.predict(features_validate)
+label_predicted_ontraindata = classifier.predict(features_train)
+
+print("Validierung: ",matthews_corrcoef(label_validate, label_predicted))
+print("Train: ",matthews_corrcoef(label_train, label_predicted_ontraindata))
+
+
+# TEST Data area
```
Comment (joh-lin) [inline Exercises/JuliusDralle/exercises_09.py:60]: Good separation of training and test dataset.

File: Exercises/JuliusDralle/exercises_09.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+df['Sex'] = df['Sex'].replace(['female','male'],[0,1])
+df['Embarked'] = df['Embarked'].replace(['S','C','Q'],[0,1,2])
+ 
+#print(df)
+
+label = df[["Survived"]]
+#features = df.drop(columns = ['Name','Ticket','PassengerId','Survived','Embarked','Cabin'])
+features = df.drop(columns = ['Name','Ticket','PassengerId','Survived','Cabin'])
```
Comment (joh-lin) [inline Exercises/JuliusDralle/exercises_09.py:47]: A little more preprocessing could have been done, maybe for cabin or ticket.

**Review by joh-lin — APPROVED** [review]

**Comment by fabischw** [comment]
I will be merging this PR if it does not get merged by the PR creator within the next 3 days in an effort to clean up the repo (More information can be found in [Issue #228](https://github.com/dhbw-ma-ppp/ppp-2023/issues/228))

**Pull request merged**